scripts/selector-example.js

```diff
@@ -53,29 +53,26 @@ Promise.all(testUUIDs.map(
 	uuid => getArticle(uuid)
 		.then(article => {
 			const $h = cheerio.load(article.bodyHTML);
-			const $x = cheerio.load(article.bodyXML);
 			const result = {uuid, title: article.title};
 
-			if($h(selector).length) result.html = $h.html(selector);
-			if($x(selector).length) result.xml = $x.html(selector);
-			if(result.html || result.xml) return result;
+			if($h(selector).length) {
+				return Object.assign({
+					html: $h.html(selector),
```

Comment:
Are we concerned that a new `.html` property being returned as part of a next-es result would now overwrite this?